tools/rack/rack.plugin/pom.xml

```diff
@@ -42,115 +42,162 @@
     <dependency>
       <groupId>org.openjfx</groupId>
       <artifactId>javafx-base</artifactId>
+      <version>${version.javafx}</version>
```

Comment:
All of these newly added versions are redundant because the parent pom already has the complete information (including versions) in its dependencyManagement section.  This allows the child pom to "inherit" the version from the parent pom.

Reply:
Removed them.